packages/astro/src/types/public/manifest.ts

```diff
@@ -0,0 +1,19 @@
+import type { AstroConfig } from './config.js';
+
+export type SerializedClientBuild = Pick<AstroConfig['build'], 'format' | 'redirects'>;
```

Comment:
Glad that these are using `Pick`. We should only be adding things to this module that we explicitly decide to add, rather than defaulting to exposing everything. If that was your thought too, mind making a comment here saying as such, so that we don't mistakenly change that in the future?